### PR TITLE
add `verification-frequency` CLI parameter

### LINKED ISSUE #646: Add fast verification freq as a CLI arg

### PR DESCRIPTION
**Pull request by darkdarkdragon**
**What does this pull request do? Explain your changes. (required)**

Adds `verification-frequency` CLI argument

- **How did you test each of these updates (required)**
Manually

**Does this pull request close any open issues?**
Fixes #646 

**Screenshots (optional):**

**Checklist:**

- [ ] I have read the **CONTRIBUTING** document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

**Review by yondonfu — APPROVED** [review]
LGTM!

File: packages/api/src/parse-cli.ts [hunk 1]
```diff
@@ -228,6 +228,11 @@ export default function parseCli(argv?: string | readonly string[]) {
         describe: "Stripe webhook secret",
         type: "string",
       },
+      "verification-frequency": {
+        describe: "verificationFreq field to return from stream/hook",
+        default: 0,
```
Comment (victorges) [inline packages/api/src/parse-cli.ts:233]: Should default be `10`, as it was before with the hardcoded value?
```suggestion
        default: 10,
```

**Review by victorges — APPROVED** [review]
LGTM

File: packages/api/src/parse-cli.ts (hunk 1 above)
Comment (darkdarkdragon) [reply packages/api/src/parse-cli.ts:233]: No, it was 10 for testing.

**Pull request merged**